lib/public/Model.js

```diff
@@ -86,10 +86,18 @@ export default class Model extends Observable {
         // Reset infinite scroll to prevent scroll listening on pages where it is disabled or not needed
         window.onscroll = null;
 
+        // Calculates the number of rows which should be visible on the page
+        const TABLEROW_HEIGHT = 45;
+        // Estimate as to how much height the navbar and other elements take up (based on the tagsOverview page)
+        const PAGE_USED_HEIGHT = 350;
```

Comment:
Currently it looks like the tables will not resize if the page size is changed.
These calculations should be part of the virtual node of the table so that they get recalculated on each page resize